### DISCUSSION TITLE
Database server had een probleem, DSMR reader komt niet meer tot leven

### DISCUSSION THREAD

**Discussion by skinkie** [dsmrreader/dsmr-reader#1462] [Questions & answers / How-to's] [answered]
Na een probleem met Postgresql (disk vol). Lukt het me niet meer om grafieken te krijgen.

![image](https://user-images.githubusercontent.com/502394/130219839-1a2fcf8d-5032-46d2-bae6-8081293754ed.png)

Ik zie het volgende gebeuren:

![image](https://user-images.githubusercontent.com/502394/130219692-93b85f38-7bd5-4f80-b8da-b6e3d6720e66.png)

![image](https://user-images.githubusercontent.com/502394/130219477-92811d58-34ef-4d6a-9666-61ec4b73fcaa.png)

Uiteraard is de schijfruimte weer beschikbaar, PostgreSQL herstart en is het uwsgi proces van dsmrreader ook herstart.

Ik heb een poging gedaan met onderstaande, maar daarmee zie ik nog geen verschil:
```bash
manage.py dsmr_stats_reconstruct_missing_day_statistics
manage.py dsmr_stats_reconstruct_missing_day_statistics_by_hours
```

**Comment by dennissiemensma** [comment]
Bedankt voor je melding. Je kunt het beste niet zomaar commands uitvoeren hoor, want die zijn bedoeld voor een hele andere situatie.

DSMR-reader loopt achter met het verwerken van alle metingen en daarom zie je geen gegevens. In je screenshot zijn dat er zo'n 43.000 en als het goed is loopt die achterstand in. De exacte snelheid hangt vooral af van de hardware.

Is de achterstand inmiddels minder of helemaal weg?

**Comment by skinkie** [comment]
Helaas niet.
![image](https://user-images.githubusercontent.com/502394/130290265-d3b64107-dcde-4f79-b825-1d08954104f4.png)

Hardware: AMD Ryzen 5 3600 (12 cores).

**Comment by dennissiemensma** [comment]
Draait het backend-proces wel? En zo ja, check dan vooral de logfile van de backend, eventueel met debug-logging ingeschakeld:
- https://dsmr-reader.readthedocs.io/nl/v4/how-to/troubleshooting/logfiles.html

**Comment by skinkie** [comment, marked as answer]
@dennissiemensma ik denk dat dat het probleem inderdaad het backend proces was. Is het mogelijk om daar een foutmelding van te geven letterlijk "het backend proces draait niet"? Op dit moment zie ik nog geen nieuwe grafieken, maar ik heb even geduld.

**Comment by skinkie** [comment]
Grafieken werken weer. Feature request blijft om aan te geven dat het backend proces niet draait.

**Comment by dennissiemensma** [comment]
Fijn om te horen. Uiteindelijk denk ik dat ik meer voel voor een stappenplan om dingen te checken, want er zijn zoveel (andere) dingen waar het aan kan liggen, dan kan iemand beter een checklist afwerken wellicht.
